tags.go

```diff
@@ -0,0 +1,16 @@
+package shopify
+
+import "strings"
+
+// Tags is a string that contains any number of short descriptors commonly used for filtering and searching, separated by commas.
+type Tags string
```

Comment:
I can't change the type of Products.Tags because that will break stuff.
This makes using tags a bit more verbose because you have to cast a bunch.

Reply:
Could still make it a breaking change though and then change it for our side in click and collect and prescription upload.